src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Net.Http.WinHttpHandler</AssemblyName>
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <!-- Although we have a NS configuration, the actual UWP implementation is a shim over WinRT: so just validate against OneCore -->
```

Comment:
>If the WinRT is just a shim why are the APIs getting flagged in that configuration?

There are no WinRT APIs in this library, System.Net.Http.WinHttpHandler.  It just has Win32 native WinHTTP API calls.

The WinRT APIs are in the System.Net.Http library package and only for the 'net50' / 'uap101' version of the binaries, i.e. for UWP.